cargo-rmc-tests/empty-main/expected.main

```diff
@@ -1 +0,0 @@
-VERIFICATION SUCCESSFUL
```

Comment:
Same here, should have been moved. This is not a big deal, but if you could change so that we avoid overriding the copyright check, that would be great.

Reply:
See the [comment below](https://github.com/model-checking/rmc/pull/198#discussion_r651023636) for why I modified this file. If I only move this `empty-main` test, then I will have to create another test (and another `*.expected` file) to check for failure and face the same copyright problem again. So I think it's better to make a decision on this problem with this PR.